Sources/Afluent/Workers/HandleEvents.swift

```diff
@@ -11,12 +11,14 @@ extension Workers {
     actor HandleEvents<Upstream: AsynchronousUnitOfWork, Success: Sendable>: AsynchronousUnitOfWork where Upstream.Success == Success {
         let state = TaskState<Success>()
         let upstream: Upstream
+        let receiveExecute: (() async throws -> Void)?
```

Comment:
My only hesitation here is the naming. `execute` is one of 2 possible ways of triggering an `AsynchronousUnitOfWork` the other is `run`.

Reply:
How would you feel about `receiveWork`?

Reply:
Could we rip off Combine and go with receiveSubscription?

Reply:
For an AUOW? I'll defer to your judgement, but IMO "subscription" feels weird for Task-related events.

Reply:
Let's go with `recieveOperation`